src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Startup.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Arcus.EventGrid.Publishing;
+using Arcus.Messaging.Tests.Core.Messages.v1;
+using Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus;
+using Arcus.Messaging.Tests.Workers.MessageHandlers;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+[assembly: FunctionsStartup(typeof(Startup))]
+
+namespace Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus
+{
+    public class Startup : FunctionsStartup
+    {
+        /// <summary>
+        /// This method gets called by the runtime. Use this method to add services to the container.
+        /// </summary>
+        /// <param name="builder">The instance to build the registered services inside the functions app.</param>
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            IConfiguration configuration =
+                new ConfigurationBuilder()
+                    .SetBasePath(builder.GetContext().ApplicationRootPath)
+                    .AddJsonFile("local.settings.json")
+                    .Build();
+            
+            builder.Services.AddTransient(svc =>
+            {
+                var eventGridTopic = configuration.GetValue<string>("Values:EventGridTopicEndpoint");
+                var eventGridKey = configuration.GetValue<string>("Values:EventGridAuthKey");
+
+                return EventGridPublisherBuilder
+                    .ForTopic(eventGridTopic)
+                    .UsingAuthenticationKey(eventGridKey)
+                    .Build();
+            });
+            
+            builder.AddServiceBusMessageRouting()
+                   .WithServiceBusMessageHandler<OrdersAzureServiceBusMessageHandler, Order>();
```

Comment:
Let's make sure we track something to have this in our templates as well

Reply:
Already here: https://github.com/arcus-azure/arcus.templates/issues/244, resolving.